environments/production.env.template

```diff
@@ -5,4 +5,7 @@ OCAML_BENCH_DOCKER_CPU=3
 OCAML_BENCH_DB_PASSWORD=
 OCAML_BENCH_GRAPHQL_KEY=
 OCAML_BENCH_TARGET_ARCH=amd64
-VITE_GRAPHQL_URL=backend_url
\ No newline at end of file
+OCAML_BENCH_HOST=
```

Comment:
Should we use `localhost` as default value for production too?

cc @rizo

Reply:
I think we should leave it for the user to decide!